app/models/miq_server/log_management.rb

```diff
@@ -165,7 +168,10 @@ def post_automate_dialogs(taskid, log_depot)
     backup_automate_dialogs(dialog_directory)
 
     logfile = LogFile.historical_logfile
-    logfile.update(:file_depot => log_depot, :miq_task => MiqTask.find(taskid))
+    logfile.update(:file_depot         => log_depot,
+                   :miq_task           => MiqTask.find(taskid),
+                   :logging_started_on => Time.zone.now,
+                   :logging_ended_on   => Time.zone.now)
```

Comment:
Maybe store these in a variable and reuse them for each value.  Does this work?  It's remarkably simple.

Reply:
`smb://[redacted]/jenkins/default_2/EVM_1/Archive_region_0_default_2_EVM_1_20200128_094413_20200129_031405.zip` 
`smb://[redacted]/jenkins/default_2/EVM_1/Current_region_0_default_2_EVM_1_20200129_113051_20200129_113156.zip` `smb://[redacted]/jenkins/default_2/EVM_1/Models_region_0_default_2_EVM_1_20200129_163211_20200129_163211.zip` 
`smb://[redacted]/jenkins/default_2/EVM_1/Dialogs_region_0_default_2_EVM_1_20200129_163212_20200129_163212.zip`